app/controllers/search_controller.rb

```diff
@@ -5,7 +5,13 @@
 class SearchController < ApplicationController
   get '/api/v1/search' do
     board_games = BoardGamesFacade.search(params['name'])
-
+ 
     BoardGameSerializer.new(board_games).serializable_hash.to_json
   end
+
+  get '/api/v1/game/categories' do
+
+    category = CategoryFacade.category
+    CategorySerializer.new(category).serializable_hash.to_json
+  end
```

Comment:
Great job extracting this endpoint and serializing. I think next steps would include refactoring this endpoint content into the `/search` endpoint as the BE app will only ever be querying that single end-point

Reply:
Yeah, I pointed out the next step of wanting to combine this data with the search endpoint in the PR, so maybe another Sinatra card can be created to support that, for the next pair to do who are working on Sinatra

Reply:
Cool! I'll create an additional card for the next steps 👍